### ISSUE TITLE
[rfp] Some rfp issues

### ISSUE THREAD

**Issue by tiagoalvesdulce** [decred/politeiagui#1877]
Recently approved proposals showing wrong status: 

<img width="742" alt="Screen Shot 2020-04-23 at 13 30 23" src="https://user-images.githubusercontent.com/13955303/80124568-d2eb0f00-8566-11ea-8e28-ff15e7243c56.png">

Need spacing between the warning and the select field:

<img width="742" alt="Screen Shot 2020-04-23 at 12 43 41" src="https://user-images.githubusercontent.com/13955303/80124708-0463da80-8567-11ea-8fc2-e30098545ff6.png">

IMO the options in this select component would be better displayed if left aligned: 

<img width="634" alt="Screen Shot 2020-04-23 at 11 59 08" src="https://user-images.githubusercontent.com/13955303/80124727-0fb70600-8567-11ea-9cc5-80b8331acdf9.png">

**Comment by fernandoabolafio** [comment]
I also found a few issues:

- All proposals in the 'in discussion' tab have the wrong 'Finished' status:
![Screenshot 2020-04-26 at 17 22 03](https://user-images.githubusercontent.com/14864439/80311940-d1c61600-87e2-11ea-81b5-20acfdd0fd77.png)

- Create a new rfp and approve it:
![Screenshot 2020-04-26 at 17 24 07](https://user-images.githubusercontent.com/14864439/80311949-dab6e780-87e2-11ea-8383-886753cb6c1b.png)

**Comment by amassarwi** [comment]
@tiagoalvesdulce @fernandoabolafio 
To test RFP work properly you should run it against https://github.com/decred/politeia/pull/1054 which wasn't merged yet.
Also, #1870 includes more RFP logic and fixes some minors on master as well, so it would be better to test RFP related stuff with #1870 state.

Anyway, I'll make sure to test/address all issues listed here on #1870.

**Comment by fernandoabolafio** [comment]
@amassarwi thanks for pointing that out. I did test it against decred/politeia#1054 though.  Perhaps some of the issues will go away in the future but it is nice to track them all.
Also, we should not merge things in the master while it is not ready in the backend yet. Otherwise, it just seem broken because the premise is that the master of politeiagui has to work without problems when pointing to latest master of politeia.

**Comment by amassarwi** [comment]
@fernandoabolafio There are unexpected new requirements for the BE PR regarding #1870 and RFPs runoff votes, we thought it will be merged sooner.
You're right, feature branch is the way to go to avoid these issues - will keep it in mind for future features.

Anyway, I'll investigate all issues listed here and will fix them if needed as part of #1870 .

**Comment by amassarwi** [comment]
@tiagoalvesdulce @fernandoabolafio 
The statuses issues seem to be resolved after luke's latest BE changes.
Other notes were addressed in #1870, except the following one:
![image](https://user-images.githubusercontent.com/10324528/81418449-00ca7900-914d-11ea-8d76-63dccca7f7dc.png)

The styling is coming from `react-select` external component that we are using in pi-ui:
```
div.customSelect__option {
    text-align: center;
    color: var(--tab-text-color);
    background-color: var(--card-background);
}
```
I don't see any simple way to override this class from `pi-ui` or `pigui` (passing classname as we do for internal components won't work here), I would suggest we rethink the usage of this external component but outside the scope of the RFP changes.

What do you think?
